Editors/PathEditor.cpp

```diff
@@ -214,10 +214,14 @@ void PathEditor::MouseReleased(Qt::MouseButton button) {
 
 void PathEditor::UpdateSelection(const QItemSelection& selected, const QItemSelection& /*deselected*/) {
   int selectIndex = -1;
+  // if new points were added to the selection, select them in the preview
   if (!selected.indexes().empty()) selectIndex = selected.indexes()[0].row();
   _ui->roomView->selectedPointIndex = selectIndex;

```

Comment:
Ideally and eventually I want these previews to share a selection model with the list views and stuff. The selected points should be the same as those selected in the list so they can be moved together logically. We should also probably be painting the selection using the current style so its consistent with platform conventions. The views should also maybe maintain a current index with focus indication like QGraphicsView/[QGraphicsScene ](https://doc.qt.io/qt-5/qgraphicsscene.html#focusItem) does.